catroid/src/main/java/org/catrobat/catroid/io/DeviceUserDataAccessor.java

```diff
@@ -136,6 +138,8 @@ public void removeDeviceValue(UserData userData) {
 				Log.e(TAG, "Device Variable File corrupted!");
 				deviceFile.delete();
 			}
+
+			Log.e(TAG, e.getMessage());
```

Comment:
Please also use Log.e with three parameters here. Maybe just add the third parameter e to the statement in line 138 and put the other Log.e in an else branch. Please check if this suggestion would work, I don't know if "Device Variable File does not exist" is the only possible error here.

```suggestion
                        else {
			     Log.e(TAG, "Device Variable File does not exist", e);
			}
```